composer/loggers/mlflow_logger.py

```diff
@@ -528,21 +608,49 @@ def log_images(
 
     def post_close(self):
         if self._enabled:
+            if hasattr(self, 'monitor_process'):
+                # Check if there is an uncaught exception, which means `post_close()` is triggered
+                # due to program crash.
+                if isinstance(self._global_exception_occurred, torch.Tensor):
+                    finish_with_exception = (self._global_exception_occurred == 1).item()
+                else:
+                    finish_with_exception = (self._global_exception_occurred == 1)
+                if finish_with_exception:
```

Comment:
@chenmoneygithub I think we can remove the torch tensor things and just use a bool for this var